site2/docs/client-libraries-java.md

```diff
@@ -602,6 +603,24 @@ The following schema formats are currently available for Java:
         .create();
   ```
 
+* Protobuf schemas can be generate using the `ProtobufSchema` class. The following example shows how to create the Protobuf schema and use it to instantiate a new producer:
+
+  ```java
+  Schema<MyProtobuf> protobufSchema = ProtobufSchema.of(MyProtobuf.class);
+  Producer<MyProtobuf> protobufProducer = client.newProducer(protobufSchema)
```

Comment:
The preferred form should be: 

```java
 client.newProducer(Schema.PROTOBUF(MyProtobuf.class))
```

Reply:
yes, I will change it. In my project the `Schema.PROTOBUF` form is used, but I wanted to keep it consistent with the sample for the JSON schema.

Reply:
Oh, I see. Do you mind also changing the example for JSON?

Reply:
sure, I will update it